src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -87,10 +87,8 @@ private BigInteger CalculateBonus(StoreView snapshot, BigInteger value, uint sta
             return value * amount * GAS.Factor / TotalAmount;
         }
 
-        internal override bool Initialize(ApplicationEngine engine)
+        internal override void Initialize(ApplicationEngine engine)
         {
-            if (!base.Initialize(engine)) return false;
-            if (base.TotalSupply(engine.Snapshot) != BigInteger.Zero) return false;
```

Comment:
We removed this check?

Reply:
`Initialize()` will be called only once. So we don't need this check.